test/workbox-webpack-plugin/node/generate-sw.js

```diff
@@ -80,10 +80,13 @@ describe(`[workbox-webpack-plugin] GenerateSW (End to End)`, function() {
       const compiler = webpack(config);
       compiler.run((webpackError) => {
         if (webpackError) {
-          expect(webpackError.message.includes('importWorkboxFrom'));
-          done();
+          if (webpackError.message.includes('importWorkboxFrom')) {
```

Comment:
FYI, in the course of writing this PR, I discovered that this test pattern wasn't actually working as it was initially implemented: it should have been `expect(webpackError.message.includes('importWorkboxFrom')).to.be.true`, although that would then cause an unhandled promise rejection rather than a clean error to report back to `mocha`.

I took the opportunity to clean up this test along with the corresponding test in the `inject-manifest.js` suite so that it properly detects the failure and reports it using the `done()` callback.